Wikipedia/Code/TalkPageViewController.swift

```diff
@@ -453,7 +459,17 @@ class TalkPageViewController: ViewController {
         if UIAccessibility.isVoiceOverRunning {
             UIAccessibility.post(notification: UIAccessibility.Notification.announcement, argument: title)
         } else {
-            WMFAlertManager.sharedInstance.showBottomAlertWithMessage(title, subtitle: subtitle, image: image, dismissPreviousAlerts: true)
+            WMFAlertManager.sharedInstance.showBottomAlertWithMessage(title, subtitle: subtitle, image: image, type: .normal, customTypeName: nil, dismissPreviousAlerts: true)
+        }
+    }
+
+    fileprivate func subscriptionErrorAlert() {
+        let title = TalkPageLocalizedStrings.subscriptionFailed
```

Comment:
If they attempt to unsubscribe and get a failure, it still says "Subscribing to the topic failed, please try again." Let's adjust so that it says "Unsubscribing to the topic failed, please try again."

Reply:
Updated!